test/integration/tape-phantom.js

```diff
@@ -1,55 +1,15 @@
+var test = require('tape')
 var Zuul = require('../../')
-
 var after = require('after')
 var assert = require('assert')
+var verify = require('./verify-common')
 
-test('tape - phantom', function (done) {
-  done = after(3, done)
-  var consoleOutput = []
-
+test('tape - phantom', function (t) {
   var config = {
-    ui: 'tape',
     prj_dir: __dirname + '/../fixtures/tape',
+    files: [__dirname + '/../fixtures/tape/test.js'],
     phantom: true,
-    concurrency: 1,
-    files: [__dirname + '/../fixtures/tape/test.js']
+    concurrency: 1
   }
-
-  var zuul = Zuul(config)
-
-  // each browser we test will emit as a browser
-  zuul.on('browser', function (browser) {
-    browser.on('start', function (reporter) {
-      reporter.on('console', function (msg) {
-        consoleOutput.push(msg.args)
-      })
-    })
-
-    browser.on('init', function () {
-      done()
-    })
-
-    browser.on('done', function (results) {
-      var endOfOutput = consoleOutput.slice(-5)
-
-      // check that we did output untill the end of the test suite
-      // this is the number of asserts in tape
-      assert.deepEqual(endOfOutput[0], ['1..9'])
-      assert.deepEqual(endOfOutput[1], ['# tests 9'])
-      assert.deepEqual(endOfOutput[2], ['# pass  5'])
-      assert.deepEqual(endOfOutput[3], ['# fail  4'])
-      assert.deepEqual(endOfOutput[4], [''])
-
-      // this is the number of passed/failed test() in tape
-      assert.equal(results.passed, 3)
-      assert.equal(results.failed, 3)
-      done()
-    })
-  })
-
-  zuul.run(function (err, passed) {
-    assert.strictEqual(err, null, 'no error')
-    assert.ok(!passed)
-    done()
-  })
+  verify(t, Zuul(config))
```

Comment:
`verify-common.js` abstracts the verification logic, which should be identical for both sauce and phantom.